src/GraphQL.Tests/Types/AutoRegisteringInterfaceGraphTypeTests.cs

```diff
@@ -417,18 +381,6 @@ public void TestBasicClassNoExtraFields()
         graphType.Fields.Count.ShouldBe(2);
     }
 
-    [Fact]
-    public async Task CustomHardcodedArgumentAttributesWork()
```

Comment:
This test also doesn't really apply since resolvers don't execute.  The attribute capability is adequately tested within AutoRegisteringObjectGraphTypeTests